### ISSUE TITLE
page: landing

### ISSUE THREAD

**Issue by ahdinosaur** [root-systems/cobuy#28]
- [x] dumb component (maybe add to storybook)
- [x] make it work with the app
- [ ] screenshot/gif grab

**Comment by ahdinosaur** [comment]
examples:

TODO ADD MORE HERE!

- http://loomio.com
- http://benthebodyguard.com/index.php
- https://camplight.net/

**Comment by sarah-arrrgh** [comment]
research ticket https://github.com/root-systems/cobuy/issues/121

**Comment by sarah-arrrgh** [comment]
New similar but different Cobuy font 
https://fonts.google.com/specimen/Lobster+Two?selection.family=Lobster+Two

**Comment by sarah-arrrgh** [comment]
End of day status:
Figuring out how to change the stupid font face and color and anything about it ever! Blocked on this but will ask for support on Monday

**Comment by sarah-arrrgh** [comment]
**FONT RESEARCH**

![image](https://user-images.githubusercontent.com/12442114/28758383-fcc1287a-75ea-11e7-9057-91645f138921.png)

![image](https://user-images.githubusercontent.com/12442114/28758382-f9473f2c-75ea-11e7-85c7-ef4928a95091.png)

![screen shot 2017-07-31 at 12 21 34 pm](https://user-images.githubusercontent.com/12442114/28758376-d883a94c-75ea-11e7-9e3a-e2915353916c.png)

REM font size calculator:
https://offroadcode.com/rem-calculator/

**Comment by sarah-arrrgh** [comment]
Points | Pixels | Ems | Percent
-- | -- | -- | --
6pt | 8px | 0.5em | 50%
7pt | 9px | 0.55em | 55%
7.5pt | 10px | 0.625em | 62.5%
8pt | 11px | 0.7em | 70%
9pt | 12px | 0.75em | 75%
10pt | 13px | 0.8em | 80%
10.5pt | 14px | 0.875em | 87.5%
11pt | 15px | 0.95em | 95%
12pt | 16px | 1em | 100%
13pt | 17px | 1.05em | 105%
13.5pt | 18px | 1.125em | 112.5%
14pt | 19px | 1.2em | 120%
14.5pt | 20px | 1.25em | 125%
15pt | 21px | 1.3em | 130%
16pt | 22px | 1.4em | 140%
17pt | 23px | 1.45em | 145%
18pt | 24px | 1.5em | 150%
20pt | 26px | 1.6em | 160%
22pt | 29px | 1.8em | 180%
24pt | 32px | 2em | 200%
26pt | 35px | 2.2em | 220%
27pt | 36px | 2.25em | 225%
28pt | 37px | 2.3em | 230%
29pt | 38px | 2.35em | 235%
30pt | 40px | 2.45em | 245%
32pt | 42px | 2.55em | 255%
34pt | 45px | 2.75em | 275%
36pt | 48px | 3em | 300%

(ref: http://reeddesign.co.uk/test/points-pixels.html)

**Comment by sarah-arrrgh** [comment]
REM vs EM - helpful article
https://zellwk.com/blog/rem-vs-em/

**Comment by sarah-arrrgh** [comment]
End of day report:
got Cobuy title font and styling working, added copy

TODO: 
- style allthethings
- change spacing around h1? (is that why the spacing is so weird above and below?)
- pretty pictures!
- how does it work at different screen sizes?

**Comment by sarah-arrrgh** [comment]
Current status:
Spending too long trying to figure out how margins and padding works with this stack. Need to drop it and do something else until @ahdinosaur can assist!

**Comment by ahdinosaur** [comment]
@sarah-arrrgh here's a demo for how to do margins and padding with the stack: https://github.com/root-systems/cobuy/commit/d5ef37e151a17cc67fef2972326b885f71e10d46. does that help?

**Comment by sarah-arrrgh** [comment]
thanks @ahdinosaur! this works in some ways, but when I try
`paddingHorizontal: theme.space[1]
`
it doesn't work. is there some special syntax for that?

**Comment by ahdinosaur** [comment]
@sarah-arrrgh `paddingHorizontal`? that'd be sweet, but in normal css (as far i understand) you have to do `paddingLeft` and `paddingRight` in order to have that desired effect.

`fela` only provides access to the [standard css property names as in the browser](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference), in camelCase format. i'm not sure what sugar-y style property names you have used before in previous projects, so can't comment on what to avoid sorry.

**Comment by sarah-arrrgh** [comment]
Thanks @ahdinosaur !

**Comment by sarah-arrrgh** [comment]
Standard CSS properties:


CSS | JavaScript
-- | --
background | background
background-attachment | backgroundAttachment
background-color | backgroundColor
background-image | backgroundImage
background-position | backgroundPosition
background-repeat | backgroundRepeat
border | border
border-bottom | borderBottom
border-bottom-color | borderBottomColor
border-bottom-style | borderBottomStyle
border-bottom-width | borderBottomWidth
border-color | borderColor
border-left | borderLeft
border-left-color | borderLeftColor
border-left-style | borderLeftStyle
border-left-width | borderLeftWidth
border-right | borderRight
border-right-color | borderRightColor
border-right-style | borderRightStyle
border-right-width | borderRightWidth
border-style | borderStyle
border-top | borderTop
border-top-color | borderTopColor
border-top-style | borderTopStyle
border-top-width | borderTopWidth
border-width | borderWidth
clear | clear
clip | clip
color | color
cursor | cursor
display | display
filter | filter
font | font
font-family | fontFamily
font-size | fontSize
font-variant | fontVariant
font-weight | fontWeight
height | height
left | left
letter-spacing | letterSpacing
line-height | lineHeight
list-style | listStyle
list-style-image | listStyleImage
list-style-position | listStylePosition
list-style-type | listStyleType
margin | margin
margin-bottom | marginBottom
margin-left | marginLeft
margin-right | marginRight
margin-top | marginTop
overflow | overflow
padding | padding
padding-bottom | paddingBottom
padding-left | paddingLeft
padding-right | paddingRight
padding-top | paddingTop
page-break-after | pageBreakAfter
page-break-before | pageBreakBefore
position | position
float | cssFloat
text-align | textAlign
text-decoration | textDecoration
text-decoration: blink | textDecorationBlink
text-decoration: line-through | textDecorationLineThrough
text-decoration: none | textDecorationNone
text-decoration: overline | textDecorationOverline
text-decoration: underline | textDecorationUnderline
text-indent | textIndent
text-transform | textTransform
top | top
vertical-align | verticalAlign
visibility | visibility
width | width
z-index | zIndex


(ref: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference)

**Comment by danalexilewis** [comment]
@sarah you could add that to the wiki :)

**Comment by sarah-arrrgh** [comment]
End of day report:

- Figured out how to add sensible spacing around things, hooray!
- Finding suitable image for landing page - took some screenshots of the Cobuy videos, pretty low res but maybe will do for now?

TODO:

- ask Lu and Beka for hi res stills of video bits
- add tagline "Helping you buy good food, together" to landing page

**Comment by sarah-arrrgh** [comment]
End of day report:

- added background image to landing page


TODO:

- [x] add tagline "Helping you buy good food, together" to landing page
- [x] style/resize background image to look good on any screen size
- [x] style/resize "Cobuy" logo to look good in relation to background image on any screen size
- [x] add "sign in" and "sign up" buttons

**Comment by sarah-arrrgh** [comment]
End of day report:

- styled background image to fit on any sized screen
- added tagline and font weights for tagline
- don't think resizing Cobuy logo is needed

TODO:

- [x] is there a way to change the opacity of the background image without affecting the inner components?
- [x] add 'sign in' and 'sign up' buttons
- [x] when screen is narrow and text is over the top of the sacks of dry goods, the text is hard to read (though this could be fixed by changing the opacity of the bg)

**Comment by sarah-arrrgh** [comment]
Background opacity link:
https://stackoverflow.com/questions/10422949/css-background-opacity/18985783#18985783

Buttons:
http://www.material-ui.com/#/components/raised-button

**Comment by sarah-arrrgh** [comment]
On exponential spacing scales:
http://tachyons.io/docs/layout/spacing/

**Comment by sarah-arrrgh** [comment]
End of day report:
The thing is did! Merged into master. Though when I ran the server from master, the bg image doesn't show up. @ahdinosaur  is this something to do with that large file thing?

TODO:
- [ ] write notes in wiki for things that might be useful for the rest of the project

**Comment by ahdinosaur** [comment]
oh, yeah probably. can you try:

```
git lfs install
git lfs pull
```

🙏

**Comment by sarah-arrrgh** [comment]
Screenshot of landing page:

<img width="938" alt="screen shot 2017-08-22 at 2 59 20 pm" src="https://user-images.githubusercontent.com/12442114/29546800-86b7d45a-874a-11e7-8c63-626b4d57ecb5.png">
